static/app/utils/issueTypeConfig/profileConfig.tsx

```diff
@@ -36,13 +36,41 @@ const profileConfig: IssueCategoryConfigMapping = {
       ),
       links: [
         {
-          text: t('File IO on Main Thread'),
+          text: t('File I/O on Main Thread'),
           link: 'https://docs.sentry.io/product/issues/issue-details/performance-issues/main-thread-io/',
         },
       ],
       linksByPlatform: {},
     },
   },
+  [IssueType.PROFILE_JSON_DECODE_MAIN_THREAD]: {
+    resources: {
+      description: t(
+        'Decoding a JSON blob into a data structure can be a long running operation, especially for large blobs, and can cause frames to drop or the app to crash. To learn more about how to fix calls that block the main thread, check out these resources:'
```

Comment:
I don't think it would likely cause a crash, maybe something like "can cause slow app performance and UI jank while scrolling"